packages/react-magma-dom/src/components/Accordion/useAccordion.ts

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { useDescendants } from '../../utils';
+
+export enum AccordionIconPosition {
+  left = 'left',
+  none = 'none',
+  right = 'right', // default
+}
+
+export interface UseAccordionProps {
+  /**
+   * Index of the accordion panel expanded by default. If multiple panels are allowed, takes an array.
+   * @default null
+   */
+  defaultIndex?: number[] | number;
+  /**
+   * Position of the chevron icon.  If 'none', the icon will not render at all.
+   * @default AccordionIconPosition.right
+   */
+  iconPosition?: AccordionIconPosition;
+  index?: number[] | number;
+  isInverse?: boolean;
+  /**
+   * If true, multiple accordion items may be expanded at once
+   * @default true
+   */
+  isMulti?: boolean;
+  /**
+   * Event that fires when an accordion button is clicked
+   */
+  onExpandedChange?: (event: any) => void;
+  testId?: string;
```

Comment:
Don't believe this is needed in the hook because it doesn't deal with rendering.

Reply:
Good call.  I'll put it in the new BaseAccordionProps interface you suggested.